producers/pipsafety/types/safety-issue.go

```diff
@@ -0,0 +1,26 @@
+package types
+
+import (
+	"encoding/json"
+)
+
+type SafetyIssue struct {
+	Name              string
+	VersionConstraint string
+	CurrentVersion    string
+	Description       string
+}
+
+//read semi-unstructured safety json into struct
```

Comment:
Comments need to start with the method name. Also should mention that you are implementing an interface. So something like:
```
// UnmarshalJSON reads semi-unstructured safety json into struct. Implements json.Unmarshaler interface.